specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -349,6 +349,36 @@
         }
       }
     },
+    "/app/subLedgerIds": {
+    "get": {
+        "operationId": "ConfidentialLedger_GetSubLedgerIds",
+        "x-ms-examples": {
+            "GetSubLedgerIds": {
+                "$ref": "./examples/ledger/GetSubLedgerIds.json"
```

Comment:
I don't see this file - maybe it didn't get committed?

Reply:
committing it now :)